extra/release_tool.py

```diff
@@ -71,7 +71,7 @@ class NotAVersionException(Exception):
 
 
 class Component:
-    COMPONENT_MAPS = None
+    COMPONENT_MAPS = {}
```

Comment:
For the error checking, I don't think it is a big difference, as an [empty dict evaluates to False in Python](https://docs.python.org/3/library/stdtypes.html#truth-value-testing). I did the change to avoid the [local] pylint warning that COMPONENT_MAPS was a "unsubscriptable" (and we first define it as None, then as a dict).

I can revert this if you wish, no strong opinion.

Reply:
My idea is that if you define it to `None`, you get exceptions if it's uninitialized, whereas if you set it to `[]`, you get silent misbehavior.

Reply:
Good point, I like that. I'll revert this and find a way to silent my pylint warning :)